### DISCUSSION TITLE
Notification page can't load

### DISCUSSION THREAD

**Discussion by Emperormummy** [community/community#143724] [Mobile] [answered]
![IMG_20241106_155345.jpg](https://github.com/user-attachments/assets/a4507bff-1090-42bc-932e-0ff4d87ae75d)

**Comment by diata0210** [comment]
This message indicates that an error occurred while executing a query. The code 9DF4:2F2742:352D30:37B3A9:672B0EAB is an error reference ID. When you contact technical support, providing this code will help them quickly locate and investigate the cause of the error.

This issue could be due to:

Unstable network connection.
Temporary server or system processing issues.
An invalid or unsupported query.
You might try reloading the page or re-executing the query. If the error persists, report it to the support team and include the reference code so they can assist you in resolving the issue.

**Reply by Emperormummy** [reply]
I think it's beta version's problem, I never seen this problem before I update my APP to 1.185.0(10223)

**Comment by Emperormummy** [comment]
> ![IMG_20241106_155345.jpg](https://github.com/user-attachments/assets/a4507bff-1090-42bc-932e-0ff4d87ae75d)
> 
> 

I think it's beta version's problem, I never seen this problem before I update my APP to 1.185.0(10223)
And I just reset the app and it still didn't solve this problem

**Comment by Emperormummy** [comment, marked as answer]
I cleared all my notifications from Github webpage, then back to APP, it fixed🧐
